apps/velo-external-db/test/drivers/wix_data_testkit.ts

```diff
@@ -0,0 +1,31 @@
+import { authConfig } from '@wix-velo/test-commons'
+const express = require('express')
```

Comment:
It is recommended to change the import to the following: 
`import express = require('express')`
That way, we won't have to provide types for the functions arguments in `app.use` 
TS will infer them by itself because now it knows what is the express variable

Reply:
`import * as express from 'express'
`
That's the way we importing it in the router, I believe it's the same.